docs/topics/modules/creating_new_project.adoc

```diff
@@ -3,20 +3,21 @@
 
 // for hello-world
 ifeval::["{context}" == "hello-world"]
-When you create a new Space, the `How would you like to get started?` menu appears. Use this menu to create a new Quickstart project as follows:
+When you create a new Space, the `How would you like to get started?` menu is displayed. Use this menu to create a new Quickstart project as follows:
```

Comment:
I'm a little confused by this one. We always aim for active voice and present tense to aid the active voice, so changing appears to is displayed seems like a step backwards.

Reply:
IBM: appear v
Do not use unless there is no better alternative. Possible alternatives include “is displayed” and “opens.”
would you prefer opens?

Reply:
Displays? As long as it's active voice and present tense, I'm fine with displays, opens, or any other option you think is more appropriate.

Reply:
Cool
@mishaone I have taken away one of the images(how would you like to get started) too, please have a look and confirm that it is alright, they were the same screenshot coming right after the other, one at the end of the previous section and second at the start of the second section.

Also request you to have a look at #119 too, very minor corrections there, please lgtm if you are ok with it.
Thanks.

Reply:
Yep, I saw that change @Preeticp. I didn't really like having them one after the other either but left them in due to modular reuse. If those modules were used elsewhere and not one after the other the screenshot would have been useful to see what we're talking about. All good with removing it, though.

Having a look at #119 now.